safe-global/web-core
# feat: Snapshot widget

## What it solves

Resolves #976

## How this PR fixes it

A basic Snapshot widget which lists the SafeDAO proposals has been created, matching the [proposed design](https://www.figma.com/file/8nKOe7h90OS7jmn3nksbq9/Governance-visibility-widgets?node-id=34%3A2394).

## How to test it

Open the dashboard and observe the new widget which matches the above design:

- Clicking a proposal should open it in a new window, as well as the "View all" and "SafeDAO Forum" links.
- Reducing the resolution should truncate the title correctly, leaving the other columns visible.

## Screenshots

![widget](https://user-images.githubusercontent.com/20442784/201058453-a11a6797-40d2-48bd-9cd2-58f80313e475.gif)

![image](https://user-images.githubusercontent.com/20442784/201058451-fa0722e3-e723-4a6c-9b2d-192dc7d327a9.png)

![image](https://user-images.githubusercontent.com/20442784/201058531-0802e2fa-c72d-4866-ba8d-b0efbb177521.png)

![image](https://user-images.githubusercontent.com/20442784/201184679-c07d09a1-e70c-413b-bb94-a04d69d6819f.png)
